dime-coding-standards/checklists/tex/sampling_randomization_checklist.tex

```diff
@@ -0,0 +1,190 @@
+\cfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/wb}}
+\rfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/analytics}}
+
+% Put checkbox to the left of the text
+\def\LayoutCheckField#1#2{% label, field
+	#2 #1%
+}
+
+% Line spacing
+\onehalfspacing
+
+% Define DIME Analytics visual identity colors
+\definecolor{fontcolor}{HTML}{7A0569}
+
+\titleformat{\section}%
+{\Large\rmfamily\bf\color{fontcolor}}% format applied to label+text
+{\llap{\colorbox{fontcolor}{\parbox{1.5cm}{\hfill\huge\color{fontcolor}\thesection}}}}% label
+{2pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\titleformat{\subsection}%
+{\large\rmfamily\color{fontcolor}}% format applied to label+text
+{}% label
+{1.5pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - Sampling and Randomization}} \\
+				\Large\textbf{{v1.0}}}
+		}
+
+  \dimeTextField{time}{2.5 cm}{Reviewer:} 
+
+        \dimeTextField{time}{2.5 cm} {Coder:}
+		
+		\textbf{NOTE:} Make sure that fill out this checklist \textbf{ONLY IF} your partner's submission includes    \textbf{sampling and/or randomization} tasks. 
+		
+		\section*{Sampling and Randomization tasks}
+  This checklist lists important factors to consider while reviewing your code review partner's \textbf{sampling/randomization} code. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+  \hfill \break 
+  \break
+  \textbf{Note:} We are piloting this checklist as part of improving the code review process. The questions in this list are therefore \textbf{not required}, and we welcome your feedback on the points listed below.
+
+  
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+   
+   
+    \item[]{\textbf{Sampling:}
+   
+            \begin{itemize}
+            
+            \item[]\dimeCheckBox{\textbf{The code clearly identifies the population eligible for sampling}. (This should be in the form of an original dataset that is fixed, unless sampling is continuous or ongoing due to research design; in that case, this should be clearly described)}
+         
+            
+            \begin{itemize}
+                \item[]\dimeCheckBox{The population dataset is uniquely identified.}
```

Comment:
```suggestion
                \item[]\dimeCheckBox{The population dataset is uniquely identified - and the code tests for it.}
```
I think it is important to include that the code tests for it. Nothing should be checked manually as then it might be true in a later re-run of the code.

Reply:
This applies to several cases below as well